### PR TITLE
2444 - JAXRS: fix user being unable to modify the EntityStream from a…

### PR DESCRIPTION
**Pull request by WhiteCat22**
… ClientRequestFilter

### CODE REVIEW

**Comment by WhiteCat22** [comment]
PB:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_hwejIClPEeivM75V9klrxw

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/bnd.overrides [hunk 1]
```diff
@@ -15,7 +15,8 @@ Export-Package: \
   org.apache.cxf.transport.https,\
   org.apache.cxf.transport.servlet,\
   org.apache.cxf.transports.http;version=3.2.1,\
-  org.apache.cxf.transports.http.configuration
+  org.apache.cxf.transports.http.configuration,\
+  org.apache.cxf.transport.stream;version=1.0.2
```
Comment (andymc12) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/bnd.overrides]: don't need this package anymore, right? `org.apache.cxf.transport.stream;version=1.0.2`

**Comment by WhiteCat22** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_jAYXgCuOEeiz4_UsMJK0Bw

**Comment by WhiteCat22** [comment]
Green PB https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_6a6O0Cu7Eeiz4_UsMJK0Bw

**Pull request merged**